cmd/kube-apiserver/app/server.go

```diff
@@ -227,6 +227,10 @@ func (s *APIServer) verifyClusterIPFlags() {
 	if s.ServiceClusterIPRange.IP == nil {
 		glog.Fatal("No --service-cluster-ip-range specified")
 	}
+	var ones, bits = s.ServiceClusterIPRange.Mask.Size()
+	if bits-ones > 20 {
```

Comment:
I'm not so familiar with Golang and don't know if the mask only appears in IPv4. If the IPNet is an IPv6 format, then just verify `ones` would be wrong since the storage is related to zeros.